lib/assets/javascripts/chef/components/icon.js

```diff
@@ -0,0 +1,24 @@
+//= require ../chef
+
+(function($, window) {
+  'use strict';
+
+  Chef.Web.Core.components.icon = {
+    init: function() {
```

Comment:
So, if you're using this method, and add new `<i class="..."></i>` icons to the DOM, they would not show up unless you called `Chef.Web.Core.components.icon.init()` again. We might want to note this somewhere.